### ISSUE TITLE
SID accuracy result inconsistent with the leaderboard.

### ISSUE THREAD

**Issue by david-wei-01001** [s3prl/s3prl#578]
Dear s3prl team, I run the experiment 
```
python3 run_downstream.py -m train -n ExpName -u hubert -d voxceleb1
```
Which gives a test accuracy of around 65%, far less than the accuracy 81% reported on the leaderboard.
This inconsistency is also reported by issue #469 

Does anybody able to reproduce the leaderboard accuracy?

**Comment by leo19941227** [comment]
Please use 1e-1 as the learning rate

**Comment by david-wei-01001** [comment]
Thanks for the reply. 

For the SID task with hubert model, in the file s3prl/upstream/hubert/hubconf.py line 94, the model used is "https://huggingface.co/s3prl/converted_ckpts/resolve/main/hubert_base_ls960.pt"

I tried to use hugging face's "facebook/hubert-base-ls960" with learning rate 1e-1 and 1e-3, and I still cannot achieve reported 80% accuracy.

What not using the hugging face's "facebook/hubert-base-ls960"? What has been changed and why do we need such a change?

Thank you very much!

**Comment by leo19941227** [comment]
Please simply use `-u hubert` with learning rate of `1e-1` or `1e-2`.
The converted checkpoint is the official checkpoint directly from fairseq toolkit and we remove the unnecessary items that depend on fairseq (like training states) and only keep the model weights so you don't need to install fairseq to make s3prl work.
On the other hand, I am not sure whether huggingface checkpoint reflects the original representation as they heavily rewrite the model definition so without fine-tuning the frozen representation might change a lot. Might need more investigation. When the SUPERB paper was published, HuBERT was not yet in Hugginface, so we build tools ourselves, and the results in SUPERB hence do not rely on Huggingface.

**Comment by david-wei-01001** [comment]
Thank you!
Could you please elaborate on how the frozen representation of hugging face transformer's implementation of hubert differ from the fairseq's implementation?

**Comment by david-wei-01001** [comment]
I have verified, they are the same. Thanks
